Hc.Gosia/Hc.Gosia.Js.Ex3/getQuestionCheckAnswear.js

```diff
@@ -0,0 +1,98 @@
+﻿var questionaire = questionaire || {};
+var actions = actions || {};
+
+(function () {
+    var points = 0;
+    var question = 1;
+    var questionCount = 10;
+    var data = {};
+
+    // get Questions
+    function getQuestions() {
+
+        actions.hideElement("start");
+        questionaire.generate(questionCount, function (questions) {
+
+            // get question and answers
+            var questionText = questions[0];
+            var answears = new Array(questionText.answears.length);
+            var isOK = new Array(answears.length);
+
+            j = 0;
+            for (i = 0; i < questionText.answears.length; i++) {
+                answears[j] = questionText.answears[i].text;
+                isOK[j] = questionText.answears[i].isCorrect;
+                j++;
+            }
+            // true position
+            var position = isOK.indexOf(true);
+
+            var data = {};
+            var data = { questionNumber: question, text: questionText.text, answears: answears, position: position }
+
+            questionaire.renderQuestion(data);
+        });
+    }
+    // render question
+    function renderQuestion(data) {
+
+        actions.showElement("center");
+        actions.hideElement("bottom");
+        //render question
+        var container = document.getElementById("header").innerHTML = "Question " + data.questionNumber + "/" + questionCount + "<br>" + data.text;
+        var options = document.getElementById("center").getElementsByTagName("input");
+
+        // render answears
+        t = 0;
+        while (t < data.answears.length) {
+            var options = document.getElementsByName("choices");
+            options[t].hidden = "";
+            document.getElementById("label" + t).hidden = "";
+            var label = document.getElementById("label" + t);
```

Comment:
możesz to przechować w zmiennej 'document.getElementById("label" + t)' przez co nie musisz szukać 2x